src/materialized/src/bin/materialized/tracing.rs

```diff
@@ -117,7 +117,7 @@ where
 pub async fn configure(
```

Comment:
Would be great to update the docco here to indicate that the log file is returned!